Spanner/src/Transaction.php

```diff
@@ -100,6 +106,16 @@ public function __construct(
         $this->context = SessionPoolInterface::CONTEXT_READWRITE;
     }
 
+    /**
+     * Get the commit stats for this transaction.
```

Comment:
We should mention that this is only available after a transaction is committed, and only if the user has requested commit stats be returned. Public methods also need samples.